.github/workflows/build.yml

```diff
@@ -27,10 +27,9 @@ on:
 env:
   BUILDX_CACHE_DIR: ~/.cache/buildx
   CURL_CACHE_DIR: ~/.cache/curl
-  IMAGE_NAME: cisagov/example
+  IMAGE_NAME: cisagov/admiral
   PIP_CACHE_DIR: ~/.cache/pip
-  PLATFORMS: "linux/amd64,linux/arm/v6,linux/arm/v7,\
-  linux/arm64,linux/ppc64le,linux/s390x"
+  PLATFORMS: "linux/amd64, linux/arm64"
```

Comment:
You should include a comment here about why the platforms were dropped and when they can be re-added.  You should also create an issue to re-add them and reference that issue in your comment.

Reply:
The comment looks good, however the commit that you reference in the #8 summary (https://github.com/cisagov/admiral-docker/pull/5/commits/58ba017966def04449ffc1455b46dd03610b4aed) is no longer correct.  You should edit it to mention https://github.com/cisagov/admiral-docker/pull/5/commits/6dc81cf4474fc2156747c0e9723cd157c198e661 instead.